modules/specifying-role-arn-albo-sts.adoc

```diff
@@ -25,7 +25,7 @@ $ oc new-project aws-load-balancer-operator
 +
 [source,terminal]
 ----
-$ cat <<EOF | oc apply -f -
+$ cat <<EOF > | oc apply -f -
```

Comment:
No No here we dont have to change, this right
Change to be done only on this doc `modules/aws-installing-an-aws-load-balancer-operator.adoc`